libs/reactive-core/src/main/java/no/nav/testnav/libs/reactivecore/utils/WebClientFilter.java

```diff
@@ -1,14 +1,48 @@
 package no.nav.testnav.libs.reactivecore.utils;
-
+import io.netty.channel.ChannelException;
+import io.netty.resolver.dns.DnsNameResolverException;
 import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.reactive.function.client.WebClientRequestException;
 import org.springframework.web.reactive.function.client.WebClientResponseException;
 
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+@Slf4j
 @UtilityClass
 public class WebClientFilter {
 
     public static boolean is5xxException(Throwable throwable) {
 
-        return throwable instanceof WebClientResponseException wce &&
-                wce.getStatusCode().is5xxServerError();
+        return (throwable instanceof WebClientResponseException wce &&
+                wce.getStatusCode().is5xxServerError()) &&
+                !wce.getStatusCode().equals(HttpStatus.INTERNAL_SERVER_ERROR) ||
+                throwable instanceof WebClientRequestException ||
+                throwable instanceof DnsNameResolverException ||
+                throwable instanceof IOException ||
+                throwable instanceof ChannelException;
+    }
+
+    public static String getMessage(Throwable throwable) {
```

Comment:
Kunne det vært nyttig å vurdere å ta i bruk [ProblemDetail](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/ProblemDetail.html)?

Reply:
Ser ut som ProblemDetail er en måte å pakke informasjonen på. Vi har allerede lagt opp til `message` og `status` her, og ProblemDetail gir vel ikke noe mer, men kan være en måte å pakke informasjonen i en ResponseEntitity for en ControllerAdvice typisk, tilsvarende ServerResponse i reaktive paradigme ... ?!